### PR TITLE
Added activity feed

### PR DESCRIPTION
**Pull request by oksim14**
Added /v2/activity-feed

Very useful :)
- small code review in send_message method

### CODE REVIEW

**Comment by oksim14** [comment]
+ small code review in send_message method

File: xboxapi/xbox_api.py [hunk 1]
```diff
@@ -102,11 +97,30 @@ def send_message(self, message, xuids=[]):
...
         return res
+
+    def send_post(self, url, data):
+        """Wrapper on the requests.post"""
+        headers = {
+            "X-AUTH": self.api_key,
+            "Content-Type": "application/json"
+        }
+
+        res = requests.post(url, headers=headers, data=data)
```
Comment (ghost) [inline xboxapi/xbox_api.py]: Can you do the `json.dumps(data)` here instead of when passing as a parameter?

**Review by ghost — CHANGES_REQUESTED** [review]
Just one minor change, but other than that looks good. Thanks!

**Comment by oksim14** [comment]
Done, json.dumps() moved to send_post method 👍

**Comment by ghost** [comment]
Thanks! TGTM

**Pull request merged**